### ISSUE TITLE
F# not able to infer correct type?

### ISSUE THREAD

**Issue by simendsjo** [fsprojects/FSharp.Control.Reactive#133]
Using `fromEventHandler`, it infers to argument to be `obj` rather then the supplied type. If I copy/paste the code to my project instead, it works, which makes me suspect something odd about the F# inference rules related to external projects or something.

https://github.com/fsprojects/FSharp.Control.Reactive/blob/master/src/FSharp.Control.Reactive/Observable.fs#L791

Maybe `_` needs to be replaced with `'EventArgs`? I haven't investigated the root cause of this.

EDIT: To reproduce

``` fshrap
    let o : IObservable<EventArgs> =
        FSharp.Control.Reactive.Observable.fromEventHandler<EventArgs>
            ignore
            ignore
```

Gives
``` example
Type mismatch. Expecting a
    'IObservable<EventArgs>'    
but given a
    'IObservable<obj>'    
The type 'EventArgs' does not match the type 'obj'
```

**Comment by PhilT** [comment]
Duplicate of #129?

**Comment by panesofglass** [comment]
Is this fixed in 4.3.0?
